packages/lib/src/components/internal/SecuredFields/lib/utilities/createIframe.ts

```diff
@@ -1,10 +1,17 @@
-export default function createIframe({ src, title = 'iframe element', policy = 'origin', styleStr = 'border: none; height:100%; width:100%;' }) {
+export default function createIframe({
+    src,
+    title = 'iframe element',
+    policy = 'origin',
+    styleStr = 'border: none; height:100%; width:100%; overflow:hidden;'
+}) {
     const iframeEl = document.createElement('iframe');
     iframeEl.setAttribute('src', src);
     iframeEl.setAttribute('class', 'js-iframe');
-    iframeEl.setAttribute('title', title);
-    iframeEl.setAttribute('frameborder', '0'); // deprecated but still necessary for IE TODO re-test this on next round of IE testing
-    iframeEl.setAttribute('scrolling', 'no');
+    if (title !== '' && title.trim().length !== 0) {
+        iframeEl.setAttribute('title', title);
+    } else {
+        iframeEl.setAttribute('role', 'presentation');
+    }
```

Comment:
Can you elaborate what the `role=presentation` attribute means on the iframe usage?

Reply:
You need to refer to the a11y spreadsheet I shared in CheckoutDevs chat: Ticket 9.1
Alternatively look at my last comment, point 1 in COWEB-1032

Reply:
Clear. Thanks!